x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authc/jwt/JwtRealmSettings.java

```diff
@@ -91,7 +98,7 @@ private JwtRealmSettings() {}
      * @return All non-secure settings.
      */
     public static Set<Setting.AffixSetting<?>> getNonSecureSettings() {
```

Comment:
I think I made getNonSecureSettings() and getSecureSettings() public because they were called directly by test code in a previous PR. The test code is gone. I can change them to private.

Reply:
I can't change getNonSecureSettings() and getSecureSettings() to private. Best I can do it is make them package private.

Those methods are calls when printing realm settings. I loop through the first to print settings destined for elasticsearch.yml, and the second to print settings for elasticsearch-keystore. The contents within each block are printed in deterministic order, because those two settings methods internally use LinkedHashMap.

Reply:
Changed to package private. Resolved.